src/placeos-rest-api/utilities/scopes.cr

```diff
@@ -48,12 +48,11 @@ module PlaceOS::Api
     end
 
     macro can_scope_access!(scope, access)
-      {% SCOPES << scope unless SCOPES.includes? scope %}
+      SCOPES << {{scope}} unless SCOPES.includes? {{scope}}
```

Comment:
This is dynamic. requiring all routes to be hit before generating a complete list.